.github/workflows/product-tests.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       - name: Maven Install
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          ./bin/retry ./mvnw install ${MAVEN_FAST_INSTALL} -pl '!presto-docs,!presto-server-rpm'
+          ./bin/retry ./mvnw package ${MAVEN_FAST_INSTALL} -am -pl presto-product-tests,presto-server
```

Comment:
you could reduce further, by cleaning ~all modules and ~/.m2 before invoking the p-t suite.

Reply:
- you need to add p-p-t-launcher to the list

- are plugins pulled by `-am` from `presto-server

Reply:
I'm thinking that simply deleting the local Maven repository before running product tests is an easier approach and will save more space. Now that you mention it, there's a Provisio issue that prevents `-am` from working (even though it gets the dependency order correct in a reactor build, this feature is somehow separate).